packages/platform-sdk-generator/src/isGothamNamespace.ts

```
@@ -14,16 +14,18 @@
  * limitations under the License.
  */
 
-export function isIgnoredNamespace(ns?: string): boolean {
+export function isGothamNamespace(ns?: string): boolean {
   switch (ns) {
-    case "Operations":
```

We should store a mapping of all foundry namespaces and gotham namespaces. If a new one is introduced, we should throw a good error explaining where the person generating needs to add. This would be to make sure we don't start generating the wrong packages for the wrong namespaces